test/val/val_cfg_test.cpp

```diff
@@ -1338,6 +1345,42 @@ TEST_F(ValidateCFG, BasicBlockIsEntryBlockOfTwoConstructsGood) {
   EXPECT_EQ(SPV_SUCCESS, ValidateInstructions());
 }
 
+// Tests that not having OpEntryPoint causes an error.
+TEST_F(ValidateCFG, NoEntryPointBad) {
```

Comment:
This isn't really a CFG test.  It's misclassified, by name certainly. Perhaps it needs to move to a different file too.